### PR TITLE
Some tests to avoid typos and missing commas

### PR DESCRIPTION
**Pull request by pjaspers**
This adds some tests for some basic sanity checking and also sets up support to run them on Travis so each PR will get the green checkmarks.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: node_js
+node_js:
+- '4.1'
+- '4.0'
+- '0.12'
+- '0.11'
+- '0.10'
+notifications:
+  slack:
+    secure: lguGyEEsQCqB1gbJPuKeWPzDLXivGgM2bflu5AJgxBHOl7N19MwiUBg46mP2qeImuSaPcGmnsAn6pjNQHgPFAuhH23T7vSyAbL7/cUIGYOBZ7teTuWKzlESirxcjNMQ0SLHRGXb959XMXspyVkO0OGP7yFz0MNnVyihTQaHQSR82ZDXuuLEFtnhh5DwVN3XjXiis7wJXXmasQN+XWyVBcuE/AUY1AV9gq6igL7IfoV86DzPHj67Z9uLAJMEANj0Z7sNVU1dkUxqtRGYBGBEX0Mxjq2hydJjekrUCOdNPIgFYS5GrNkgtK1d0TC5oobn8YHPmQ51Pcd70UPenP8K6mvCEm2xR4uPvBaSIIozn4SJDSYkGzqZQXNZDoXo2edk1MlE3NHsbdui9bZo7wJ+5KOLScKKwFMS6c4SGOxLY1OTChnvQgzjbDVEeCxEamlZwfizsy6Yrehf2W7cV2hyJpVQPDhlNzFgq8eEk1RJRjqRWcKJshQ4iFGeV38sZ+VPeeV88CdXnAQx4579enP1TMq56Y+QnaPiqkg3TZ1JjKkO4LdhMzGEUFK/JIrJdakcQZDwLZiFuCwPdKAHPUaGsXCXMQTO6RHsCfo8pPFQz0Va8sUWmvdr8rfFjCCaW/05Hxf1NjiIPI2lqV5RouV7OPbWz3hNSvY7tUcUULvtKY5o=
```
Comment (Reprazent) [inline .travis.yml:10]: Isn't this a public repo?

Comment (pjaspers) [inline .travis.yml:10]: Isn't this a secure entry?

Comment (Reprazent) [inline .travis.yml:10]: Don't know, just wanted you to think about this being public, if you say it's ok it must be ok...

## 

Bob Van Landuyt
+32 473 43 05 97

Comment (pjaspers) [inline .travis.yml:10]: [More info](http://docs.travis-ci.com/user/encryption-keys/) 

It's basically encrypted on a per repo basis, so I'm wondering if this key will still work once this gets pulled into the repo. (I'd say no)

Comment (Reprazent) [inline .travis.yml:10]: Cool!

Only one way to find out.

## 

Bob Van Landuyt
+32 473 43 05 97

**Pull request merged**